replset-consistency/README.md

```diff
@@ -246,6 +246,13 @@ When remediation is complete, resume writes to the collection(s) being remediate
 
 ## 5. Resolve any remaining index inconsistencies
 
+If you are remediating for local inconsistencies only (only ran validate):
```

Comment:
Instead of "(only ran validate)" can we anchor link back to "Checking for Local Inconsistencies"